DigitalLearningSolutions.Web/Views/Frameworks/Developer/ResourceSearch.cshtml

```diff
@@ -0,0 +1,22 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Common.SearchablePage
+@model BaseSearchablePageViewModel
+
+<form method="get" role="search">
+  @if (Model.FilterEnabled)
+  {
+    <input type="hidden" name="filterBy" value="@Model.FilterBy" />
+  }
+  @foreach (var (key, value) in Model.RouteData)
+  {
+    <input type="hidden" name="@key" value="@value" />
+  }
+  <input type="hidden" name="sortBy" value="@Model.SortBy" />
+  <input type="hidden" name="sortDirection" value="@Model.SortDirection" />
```

Comment:
or this. But we may want to add  `<input type="hidden" name="limit" value="10" />` for when we call the LH OpenAPI search endpoint.